pams/sim/g2t/g2t_stg.F

```diff
@@ -14,42 +14,43 @@ FUNCTION  G2T_STG (g2t_track_h,   g2t_track ,
       g2t_fts_hit_h.nok = 0
 
 
-      "The geometry contains deprecated sensitive volume FSTA."
-      "Only setup readout for volume TGCG"
-
-      CALL g2r_get_sys('STGM','TGCG',Iprin,Idigi)      
-      IF (Iprin.lt.0) go to 99
-
-       Do While (G2R_GET_HIT('stg') .eq. 0)
-*         print *,i,id,trac,next,volume,
-*    >            x(1),x(2),x(3),xx(1),xx(2),xx(3),c(1),c(2),c(3),
-*    >            p(1),p(2),p(3),p(4),r,rr,phi,the,eta,tdr,tof,
-*    >            Slen,Step,Etot,Lgam,Lpto,Elos,User,Unkn(3)
-
-         i=i+1
-         g2t_fts_hit_h.nok            = i
-         g2t_fts_hit(i).id            = i
-
-           g2t_fts_hit(i).x(1)        = xx(1)!rr*cos(phi)
-           g2t_fts_hit(i).x(2)        = xx(2)!rr*sin(phi)
-           g2t_fts_hit(i).x(3)        = xx(3)
-           g2t_fts_hit(i).p(1)        = p(1)
-           g2t_fts_hit(i).p(2)        = p(2)
-           g2t_fts_hit(i).p(3)        = p(3)
-
-         g2t_fts_hit(i).tof           = tof
-         g2t_fts_hit(i).de            = Elos
-         g2t_fts_hit(i).ds            = Step
-         g2t_fts_hit(i).track_p       = trac
-         g2t_fts_hit(i).volume_id     = volume
```

Comment:
I presume that the overall structure of the detector remains the same... i.e. the paths to the sensitive volumes are the same, modulo the change in the names of the volumes.  But there are now   three SDs where previously there was only one.  So, does the g2t_volume_id encoding need to change? (pams/sim/g2t_volume_id.g).